components/neutron/aio-values.yaml

```
@@ -11,6 +11,13 @@ endpoints:
       name: rabbitmq-server
     hosts:
       default: rabbitmq-nodes
+  network:
+    port:
+      api:
```

```suggestion
      server:
```

I think neutron uses this

---

hmm where did you see that?
[The openstack-helm values.yaml for neutron](https://opendev.org/openstack/openstack-helm/src/commit/8e5ad85fd09983e07ddc87de76e480b4ceca9120/neutron/values.yaml#L2535) seems to have it as 'api'